editor/plugins/org.fusesource.ide.wsdl2rest.ui/pom.xml

```diff
@@ -83,8 +83,13 @@
 			<version>${version.wsdl4j}</version>
 		</dependency>
 		<dependency>
-			<groupId>commons-lang</groupId>
-			<artifactId>commons-lang</artifactId>
+		    <groupId>commons-lang</groupId>
+		    <artifactId>commons-lang</artifactId>
+		    <version>2.6</version>
```

Comment:
it seems wsdl2rest is using version 2.4.
I think it will be best to extract it as constant like for other dependencies

![image](https://user-images.githubusercontent.com/1105127/57068347-bf012a00-6cd1-11e9-896e-9d0f2e112caa.png)